### DISCUSSION TITLE
VS CODE : error because C:\/.git doesn't exist

### DISCUSSION THREAD

**Discussion by Donfox95** [JanDeDobbeleer/oh-my-posh#3138] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

i'am using VS studio Code  with integrated terminal getting following error : 
 
getting an error because git / git posh / oh my posh cannot find C:\/.git: 

![ohmyposh](https://user-images.githubusercontent.com/106558878/204241763-3bba4638-7f83-486a-bcc5-6393c487bc89.PNG)

### Theme

default

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell, other (please specify)

### Log output

```Shell
2022/11/28 10:23:00 Pwd
C:\Users\admin.pieterJ
2022/11/28 10:23:00 Pwd: 0s
2022/11/28 10:23:00 HasParentFilePath
CreateFile C:\/.git: Het systeem kan het opgegeven bestand niet vinden. ( translation : cannot find path but this is normal ) 
2022/11/28 10:23:00 HasParentFilePath: 19.1104ms, args: .git
2022/11/28 10:23:00 Pwd
C:\Users\admin.pieterJ
2022/11/28 10:23:00 Pwd: 0s
2022/11/28 10:23:00 Home
C:\Users\admin.pieterJ
2022/11/28 10:23:00 GOOS: 0s
2022/11/28 10:2
```

**Comment by JanDeDobbeleer** [comment]
@Donfox95 looking at the error message in your terminal, can you please [upgrade first](https://ohmyposh.dev/docs/migrating) and see if the error still persists? Thanks.

**Comment by Donfox95** [comment]
![ohmyposh2](https://user-images.githubusercontent.com/106558878/204272102-fa649532-94c3-4434-bc38-07c9098c4137.PNG)


as you see in following screenshots

in terminal it works but in visual studio code it doesn't :( 


`
C:\Program Files\Git\cmd\git.exe
2022/11/28 12:53:11 CommandPath duration: 3.5814ms, args: git.exe
2022/11/28 12:53:11 HasCommand duration: 3.5814ms, args: git.exe
2022/11/28 12:53:11 debug: Pwd
C:\Users\Pieter-jan.devos
2022/11/28 12:53:11 Pwd duration: 0s, args:
2022/11/28 12:53:11 error: HasParentFilePath
CreateFile C:\.git: Het systeem kan het opgegeven bestand niet vinden.
2022/11/28 12:53:11 HasParentFilePath duration: 2.5679ms, args: .git
2022/11/28 12:53:11 debug: Pwd
C:\Users\Pieter-jan.devos
2022/11/28 12:53:11 Pwd duration: 0s, args:
2022/11/28 12:53:11 debug: Home
`
there is still same error message in the error logs

**Reply by JanDeDobbeleer** [reply]
@Donfox95 are you sure the error comes from oh-my-posh as this is a PowerShell error. Can you try to also upgrade PSReadLine? Alternatively, disable shell integrations in vscode as they're known to be buggy. There's no custom logic for code in oh-my-posh so this is most likely caused by something else.
